### DISCUSSION TITLE
build trouble

### DISCUSSION THREAD

**Discussion by verkaro** [RealMelkor/Gemigit#5] [General]
I am currently running go 20.3. After doing a git pull today and trying to build I get this:

$ go build
go: github.com/gabriel-vasile/mimetype@v1.4.0: missing go.sum entry; to add it:
        go mod download github.com/gabriel-vasile/mimetype
go: github.com/gabriel-vasile/mimetype@v1.4.0: missing go.sum entry; to add it:
        go mod download github.com/gabriel-vasile/mimetype

I don't have enough insight into how go works to offer a suggestion. I've compiled a few other go programs this morning and they all worked.

**Comment by RealMelkor** [comment]
I just updated the go.mod and go.sum files, it should work now. I think the problem was caused by the addition of a new dependency and those 2 files didn't get pushed.
